src/playback/qml/MuseScore/Playback/internal/AuxSendControl.qml

```diff
@@ -73,6 +73,29 @@ Item {
             onNewValueRequested: function(newValue) {
                 root.auxSendItemModel.audioSignalPercentage = newValue
             }
+
+
+            states: [
```

Comment:
Let's use the same approach as in FlatButton, the accentButton property

It should look like `accentControl: root.auxSendItemModel.isActive` after the changes.

It's essential to have consistent approaches to component interfaces to make them easy to use in the future.

Reply:
I think it should be done even differently: we should use the `enabled` property, because interaction needs to be blocked too, it's not just about colours, if I understand correctly.
```qml
    enabled: root.auxSendItemModel.isActive
```
And then, inside KnobControl, apply the correct colors based on the value of the `enabled` property.

Reply:
@jessjwilliamson should we prohibit the user from interacting with the KnobControl?

Reply:
Not sure if I understand the question 100% Prohibit interaction while disabled? No. User should still be able to change values even when bypassed

Reply:
Oh, interesting... Then I said nothing :)

Reply:
No worries! :D